staging/src/k8s.io/apiserver/pkg/server/resourceconfig/helpers.go

```diff
@@ -98,6 +106,11 @@ func MergeAPIResourceConfigs(
 			continue
 		}
 		groupVersionString := tokens[0] + "/" + tokens[1]
+		// HACK: Hack for "v1" legacy group version.
```

Comment:
can we avoid leaking this here and maybe do the conversion before in kube-apiserver?

Reply:
Done